<launch>
  <node type="point_cloud2_assembler" pkg="laser_assembler"
        name="assembler">
    <remap from="cloud" to="cloud_out"/>
    <param name="max_clouds" type="int" value="600" />
    <param name="fixed_frame" type="string" value="mur620a/base_frame" />
    <param name="tf_cache_time_secs" type="double" value="10" />
    <param name="/use_sime_time" value="true"/>
  </node>
</launch>